### PR TITLE
Dev aek chapter1

### PR DESCRIPTION
**Pull request by Lenardgeorge**
- Added a introductory section to chapter 1, section 1.3 
- Added cover images to all the sections
- Updated the step by step of the Building the rover

### CODE REVIEW

File: content/Carla-Fabricio/newRecipeFT/New document.md [hunk 1]
```diff
@@ -4,7 +4,11 @@ This is gonna be the best buttermilk pancake that you have ever see
...
 ***
 
-![cover](C:\Users\Arduino employee\Documents\Arduino Github\musical-pancake\content\Carla-Fabricio\images\cover.jpeg)
+
+
+![BEST BUTTERMILK PANCAKE EVER](images/recipeCover.jpg)
+
+
 
 * Toons of buttermilk
```
Comment (GDJose) [inline content/Carla-Fabricio/newRecipeFT/New document.md:13]: It is gonna be  super healthy

File: content/Carla-Fabricio/newRecipeFT/New document.md [hunk 2]
```diff
@@ -4,7 +4,11 @@ This is gonna be the best buttermilk pancake that you have ever see
...
 * Vanilla extract  as vanilla as the Scandinavians
```
Comment (GDJose) [inline content/Carla-Fabricio/newRecipeFT/New document.md:14]: this is racist  ;P

File: content/Carla-Fabricio/newRecipeFT/New document.md (hunk 1 above)
Comment (Lenardgeorge) [reply content/Carla-Fabricio/newRecipeFT/New document.md:13]: what do I need to add to make it unhealthy

File: content/Carla-Fabricio/newRecipeFT/New document.md [hunk 3]
```diff
@@ -4,7 +4,11 @@ This is gonna be the best buttermilk pancake that you have ever see
 
 ***
 
-![cover](C:\Users\Arduino employee\Documents\Arduino Github\musical-pancake\content\Carla-Fabricio\images\cover.jpeg)
+
+
+![BEST BUTTERMILK PANCAKE EVER](images/recipeCover.jpg)
```
Comment (Lenardgeorge) [inline content/Carla-Fabricio/newRecipeFT/New document.md:9]: THIS WORKS

**Pull request merged**